### PR TITLE
update #74895

### PR DESCRIPTION
**Pull request by wout-codes**
#74895

### CODE REVIEW

File: FrenchFilter/sections/specific.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 !-------------------- Specific rules --------------------!
 !--------------------------------------------------------!
 !
-rojadirecta.fr,veopartidos.net##.overlay
+veopartidos.net,lowend.xyz###overlay
```
Comment (zloyden) [inline FrenchFilter/sections/specific.txt:5]: `veopartidos.net` is dead on my end. Remove it.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: FrenchFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply FrenchFilter/sections/specific.txt:5]: @zloyden It's the player in the website.

Comment (zloyden) [reply FrenchFilter/sections/specific.txt:5]: Something is wrong with my VPN. Ok.

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
@zloyden They changed their code just now. Add this directly `||jokingst.xyz^$third-party`

**Pull request merged**